Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/ConfigBean.java

```diff
@@ -82,7 +84,8 @@ public ConfigBean(String siteEditor, String siteExternalHost, int siteExternalPo
                       boolean enableUserReg, boolean enableFullUserReg, boolean enableSubscription,
                       boolean enableCalendar,
                       long maxUploadSize,
-                      String contactEmail) {
+                      String contactEmail,
+                      boolean useKeycloakSso) {
 
         LoggerFactory.getLogger(getClass()).info("Portal configured with:" +
```

Comment:
Since this logger has lingered for a few years, just upgrade it to being a static final field.

Reply:
Made consistent, removed years of lingering logger to be consistent with other classes (top of class static) declaration [DONE].